src/devices/Neo-M8/Neo-M8.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!--Disabling default items so samples source won't get build by the main library-->
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="*.cs" />
+    <None Include="README.md" />
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+    <ProjectReference Include="../Nmea0183/Nmea0183.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.IO.Ports" Version="4.6.0-preview8.19405.3" />
```

Comment:
Do you actually need the latest preview for this? Would it be possible to just use latest stable? I ask because we are not yet sure if we are going to ship at the same time of .NET Core or a little bit earlier, so it would be good if we don't depend on preview versions if possible.

Reply:
but this is OOB package, right? we need a version from at least around January this year so I think it will be preview either way

Reply:
I see, bummer. That's fine then, we'll have to ship after .NET Core then.